Volume/Views/PublicationList.swift

```diff
@@ -46,21 +44,31 @@ struct PublicationList: View {
         NavigationView {
             ScrollView(showsIndicators: false) {
                 followedPublications
+                Spacer()
+                    .frame(height: 48)
                 notFollowedPublications
             }
-            .navigationTitle("Publications.")
+            .background(Color.volume.backgroundGray)
+            .toolbar {
+                ToolbarItem(placement: ToolbarItemPlacement.navigationBarLeading) {
+                    BubblePeriodText("Publications")
+                        .font(.begumMedium(size: 24))
+                        .offset(y: 8)
+                }
+            }
+            .navigationBarTitleDisplayMode(.inline)
         }
     }
 }
 
-// TODO: replace w/ Volume specific design
-private struct Header : View {
-    @State var text: String
-    
-    var body : some View {
-        Text(text)
-            .frame(maxWidth: .infinity, alignment: .leading)
-            .padding(.leading)
+extension PublicationList {
+    func header(_ text: String) -> some View {
+        HStack {
+            UnderlinedText(text)
+                .font(.begumMedium(size: 20))
+            Spacer()
+        }
+        .padding(.leading)
```

Comment:
this extension will need to go into a separate file (and become struct) once we merge with the other PR since we need this header in both `PublicationList`, `HomeList`, and `BookmarkList`.